clients/cpp-console/RaspberryPi.md

```diff
@@ -40,12 +40,13 @@ There are many ways to do development on a Raspberry pi. It may be useful to tak
 
   ```sh
   ln -s Cognitive-Services-Voice-Assistant/samples/clients/cpp-console
+  ln -s Cognitive-Services-Voice-Assistant/samples/keyword-models
```

Comment:
this link will work, but the deployment script for the GGEC needs to be updated since it assumes no links. Should be a simple path replacement

Reply:
@chschrae Can you point me to the script? I can do the change, but I won't be able to test it right now